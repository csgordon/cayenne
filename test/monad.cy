module example$monad = 
#include Prelude
let xys = do Monad_List
             (x::Integer) <- 2 : (3 : Nil)
             (y::Integer) <- 4 : (5 : Nil)
             return (x*y)
in  putStrLn (System$List.show show xys)

