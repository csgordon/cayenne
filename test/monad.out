8 : 10 : 12 : 15 : Nil
